internal-packages/workflow-designer-ui/src/editor/tool/toolbar/toolbar.tsx

```diff
@@ -645,49 +525,45 @@ export function Toolbar() {
 												onOpenAutoFocus={(e) => {
 													e.preventDefault();
 												}}
+												onMouseEnter={handleCapabilityPanelEnter}
+												onMouseLeave={handleCapabilityPanelLeave}
 											>
 												<div className="absolute z-0 rounded-[8px] inset-0 border mask-fill bg-gradient-to-br from-[hsla(232,37%,72%,0.2)] to-[hsla(218,58%,21%,0.9)] bg-origin-border bg-clip-boarder border-transparent" />
+												{/* Pro plan overlay for free users */}
+												{languageModelMouseHovered &&
+													isProModelForFreeUser(
+														languageModelMouseHovered,
+														userTier,
+													) && (
+														<div
+															role="tooltip"
+															className="absolute inset-[2px] z-10 bg-black/80 backdrop-blur-sm rounded-[6px] flex items-center justify-center"
+														>
+															<div className="text-center">
+																<p className="text-blue-400 text-[14px] font-medium mb-3">
+																	This model is available on Pro plan
+																</p>
+																<button
+																	type="button"
+																	className="text-blue-400 hover:text-blue-300 border border-blue-400 hover:border-blue-300 text-[12px] px-4 py-2 rounded-[6px] font-medium transition-colors"
+																	onClick={() => {
+																		window.open("/pricing", "_blank");
```

Comment:
```suggestion
																		window.open("/settings/team", "_blank");
```

Reply:
@kaochannel154 
The `/pricing` page does not exist, so could you please change it to `/settings/team` for now?